packages/contracts/markets/base/reservesConfigs.ts

```diff
@@ -23,6 +24,36 @@ export const strategyUSDbC: IReserveParams = {
   reserveFactor: ethers.utils.parseUnits('0.1',18).toString(),
 };
 
+export const strategyUSDC: IReserveParams = {
+  strategy: rateStrategyStableThree,
+  baseLTVAsCollateral: ethers.utils.parseUnits('0.77',18).toString(),
```

Comment:
Where did you get these risk params from?

Reply:
extrapolated from usdbc